### DISCUSSION TITLE
can't connect to sever error code:4214

### DISCUSSION THREAD

**Discussion by thegreatestbrian** [Melledy/Grasscutter#1521] [Support]
everything is running very well until this error showing up, i'm stuck at this, someone can solve this?

**Comment by dogedeathhaha** [comment]
Me too

**Comment by sugeinmy** [comment]
i'm using 2.8.5 test client ,and found the same 4214 problem

**Comment by script-M** [comment]
Error code: 4214. Generally, the reason is that the client version cannot match the server version

**Comment by cnlancehu** [comment]
in version 2.8 the genshin added validation for domain name resolution, you need to unzip this patch and replace the files in the game directory
download from here https://file.lancex.top/genshin%20patch.zip

**Reply by flswld** [reply]
not effect, use this metadata file in 2.8.50 client, it was directly crash when launcher

**Reply by cnlancehu** [reply]
but i can launch the game correctly,did u replace all the files to the game directory?
